policy-service/src/main/resources/application.yaml

```diff
@@ -48,3 +48,9 @@ management:
         - loggers
     loggers:
       enabled: true
+
+cache:
+  policyWarmer: stdPolicyCacheWarmer
```

Comment:
See comment on ApplicationConfiguration...
can this become:
```
populationConfig:
    policy: std
    user: std
```
?

Reply:
Updated